crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -70,14 +70,15 @@ impl Violation for DocstringMissingReturns {
 }
 
 /// ## What it does
-/// Checks for function docstrings that have a "returns" section without
-/// needing one.
+/// Checks for function docstrings with redundant "Returns" sections.
```

Comment:
Somehow "redundant" makes it seem like they had more than one Returns section or something.

```suggestion
/// Checks for function docstrings with unnecessary "Returns" section.
```

Reply:
I think it should be _sections_ rather than _section_, though, since we're talking about "function docstrings", which is plural?

Reply:
I defer to the crown for matters linguistic 👑